client/src/Components/AccountTransactions.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import Grid from '@material-ui/core/Grid';
+import Paper from '@material-ui/core/Paper';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import List from '@material-ui/core/List';
+import PropTypes from 'prop-types';
+
+const AccountTransactions = props => {
+  return (
+    <>
```

Comment:
Just a heads up, you don't need to use a fragment (`<>`) here, because you're not wrapping more than one component.